Whiskey/Tasks/Invoke-WhiskeyExec.ps1

```diff
@@ -11,6 +11,8 @@ function Invoke-WhiskeyExec
 
     Pass arguments to the executable via the `Argument` property. The `Exec` task uses PowerShell's `Start-Process` cmdlet to run the executable, so that arguments will be passes as-is, with no escaping. YAML strings, however, are usually single-quoted (e.g. `'Value'`) or double-quoted (e.g. `"Value"`). If you're using a single quoted string and need to insert a single quote, escape it by using two single quotes, e.g. `'escape: '''` is converted to `escape '`. If you're using a double-quoted string and need to insert a double quote, escape it with `\`, e.g. `"escape: \""` is converted to `escape: "`. YAML supports other escape sequences in double-quoted strings. The full list of escape sequences is in the [YAML specification](http://yaml.org/spec/current.html#escaping in double quoted style/).
 
+    The `Exec` task supports a simplified single line syntax to define the `Path` and optional `Arguments` properties. Anything enclosed by single-quote or double-quote characters are treated as an individual path or argument. Otherwise, white-space is the default delimiter separating items.

```

Comment:
Give an example.

```
BuildTasks:
- Exec: powershell.exe -NonInteractive -NoProfile -ExecutionPolicy Bypass $PSVersionTAble
```

Reply:
I previously added an Example #3. Did you have something else in mind or just overlooked? :)